src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -197,7 +197,7 @@ public DirectShapeRoomBoundingOption(IEnumerable<PortModel> inPorts, IEnumerable
     [OutPortTypes("string")]
     public class DetailLevel : CustomGenericEnumerationDropDown
     {
-        private const string outputName = "Detail Level";
```

Comment:
Why are we changing the node output name to lowercamel case here?

Reply:
@reddyashish this was to match the names of the output ports provided by Helena
![image](https://github.com/DynamoDS/DynamoRevit/assets/43077096/f57a4876-9669-4ffd-8088-9cc753409b8f)
https://www.figma.com/file/4JFNaoH1YdbEh9tXQ9JRB6/DynamoRevit-nodes?type=design&node-id=0-1&t=adSKSuTxnt8rxXcg-0